### PR TITLE
TFLint fixes

### PR DESCRIPTION
**Pull request by tamirkamara**
Various TFLint fixes

### CODE REVIEW

File: templates/shared_services/admin-vm/terraform/admin-jumpbox.tf [hunk 1]
```diff
@@ -29,7 +29,7 @@ resource "azurerm_windows_virtual_machine" "jumpbox" {
   resource_group_name        = data.azurerm_resource_group.rg.name
   location                   = data.azurerm_resource_group.rg.location
   network_interface_ids      = [azurerm_network_interface.jumpbox_nic.id]
-  size                       = var.admin_jumpbox_vm_sku
+  size                       = "XXX"
```
Comment (anatbal) [inline templates/shared_services/admin-vm/terraform/admin-jumpbox.tf:32]: Is that working?

Comment (tamirkamara) [reply templates/shared_services/admin-vm/terraform/admin-jumpbox.tf:32]: It's meant to test the linter and see if it works correctly. I'll revert this when done testing.

**Review by damoodamoo — APPROVED** [review]

**Pull request merged**